### DISCUSSION TITLE
Accessing the standard form of the LP

### DISCUSSION THREAD

**Discussion by kartinator89** [coin-or/Clp#170] [Q&A]
Is there a way to access the standard form of the Linear Program after reading in a .lp file. i.e., the model after slack/surplus variables are added to convert all constraints to equalities with positive RHS and after free variables are handled using non-negative variables?

**Comment by jajhall** [comment]
No, it's not. Efficient simplex solvers like Clp don't carry out any of the transformations that you list. The code handles fully general bounds on variables and constraints.
